### PR TITLE
[hotfix] fixes app freeze, when password widget drops inside a listview

### LINKED ISSUE #3831: Dropping password widget inside a listview freezes the app
### What is the expected behaviour?
No unusual behaviour should be seen when widgets are dropped inside Layout widgets(Listview here).

### What is the current behaviour?
Dropping the password widget inside a listview freezes the app

### How to reproduce the issue?

1. Drag and drop a listview widget from the widget manager
2. Drag and drop a **password** widget inside the listview widget

### PR DESCRIPTION
**Pull request by arpitnath**
Fixes #3831

### CODE REVIEW

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@arpitnath I'm not able to see the `isValid` exposed value in the inspector.

image

**Review by kavinvenkatachalam — APPROVED** [review]
LGTM

**Comment by kavinvenkatachalam** [comment]
@akshaysasidrn @gondar00 Can you check this PR?

**Review by gondar00 — APPROVED** [review]
LGTM

**Comment by Nandinisaha13** [comment]
https://tooljet.qatouch.com/v2#/case/list/p/xVP5/llJwn/er6RP - Test Case

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "prebuild:plugins:prod": "npm run install:plugins",
     "build:plugins:prod": "NODE_ENV=production npm run build:plugins",
     "postbuild:plugins:prod": "npm --prefix plugins prune --production",
-    "prebuild:frontend": "npm --prefix frontend install",
+    "prebuild:frontend": "npm --prefix frontend install --production=false",
```
Comment (sherfin94) [inline package.json:48]: Hey @arpitnath, was wondering why we need this change? It seems to have introduced a lot of package changes.

Comment (arpitnath) [reply package.json:48]: @gondar00

Comment (gondar00) [reply package.json:48]: This change is related to Heroku build size being more than 500MB after adding a few packages, which I have explained in detail in this issue [#3797](https://github.com/ToolJet/ToolJet/issues/3797)

**Comment by arpitnath** [comment]
Ref: #4018

**Pull request closed without merging**